packages/twenty-front/src/modules/workflow/hooks/useFindShowPageWorkflow.tsx

```diff
@@ -0,0 +1,9 @@
+import { useFindWorkflowWithCurrentVersion } from '@/workflow/hooks/useFindWorkflowWithCurrentVersion';
+import { showPageWorkflowIdState } from '@/workflow/states/showPageWorkflowIdState';
+import { useRecoilValue } from 'recoil';
+
+export const useFindShowPageWorkflow = () => {
```

Comment:
Same, we should not put `Find` neither `ShowPage`. I would remove that hooks completely. This is only one additional call to recoil to do in the parent component directly and then call `useWorkflowWithCurrentVersion`

Reply:
Done